@navikt/ds-css/grid/container.css

```diff
@@ -0,0 +1,13 @@
+.navds-grid-container {
+  padding: var(--navds-grid-gutter-small);
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1272px;
```

Comment:
bruke rem istedenfor px for avstander? Så skalerer det bedre 🤗 Ser det gjelder flere steder. Men i media-query maker det vel sense å bruke px